connectors/kafka/pom.xml

```diff
@@ -31,16 +31,24 @@ except in compliance with the proprietary license.</license.inlineheader>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.danilopianini</groupId>
+      <artifactId>gson-extras</artifactId>
```

Comment:
Good point. I deleted this particular library, but maybe it would be better to replace Gson with Jackson as a separate task, since this is now working with Gson. Of course I can do it as part of this task, we just have to decide how urgent this task is. WDYT?